### PR TITLE
Doc clarifications

### PR DESCRIPTION
**Pull request by NickGoog**
Addresses a few bugs we've received.

### CODE REVIEW

File: gslib/commands/mb.py [hunk 1]
```diff
@@ -125,7 +125,7 @@
   retention policy.
 
 <B>UNIFORM BUCKET-LEVEL ACCESS</B>
-  You can specify one of the available settings for a bucket
+  You can specify turn on uniform bucket-level access for a bucket
```
Comment (starsandskies) [inline gslib/commands/mb.py:128]: Suggestion: "You can enable or disable uniform bucket-level access for a bucket with the -b option"

File: gslib/commands/rb.py [hunk 2]
```diff
@@ -51,6 +51,8 @@
   -f          Continues silently (without printing error messages) despite
               errors when removing buckets. If some buckets couldn't be removed,
               gsutil's exit status will be non-zero even if this flag is set.
+              If no buckets could be removed, the command will raise a
```
Comment (starsandskies) [inline gslib/commands/rb.py:54]: "will raise" --> "raises"

**Review by starsandskies — APPROVED** [review]
Two nits, but overall LGTM!

File: gslib/commands/mb.py (hunk 1 above)
Comment (NickGoog) [reply gslib/commands/mb.py:128]: Done

File: gslib/commands/rb.py (hunk 2 above)
Comment (NickGoog) [reply gslib/commands/rb.py]: Done

**Pull request merged**